src/nimib/renders.nim

```diff
@@ -20,6 +20,8 @@ proc useHtmlBackend*(doc: var NbDoc) =
   doc.partials["nbCapture"] = """{{>nbCodeOutput}}"""
   doc.partials["nbCodeSource"] = "<pre><code class=\"nohighlight hljs nim\">{{&codeHighlighted}}</code></pre>"
```

Comment:
should this line and the following be removed?

Reply:
(from here it is not clear but later lines seem to override this)

Reply:
Correct :+1: Will delete them